src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/ProductPreferences/PaginationType.php

```diff
@@ -70,6 +70,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'Brand' => 5,
                     'Product quantity' => 6,
                     'Product reference' => 7,
+                    'Product sales' => 8,
```

Comment:
```suggestion
                    'Product best sales' => 8,
```
The options are passed hard-coded, does it mean they are never translated?

Reply:
Ah they are thanks to the `translation_domain` option below But there's another problem then if this wording is not used somewhere else when used with the `trans` method it will never be extracted

Reply:
To make sure all these wordings are correctly scanned you should use `$this->trans` to generate each key

Reply:
@jolelievre It will work, the scan tool somehow discovers these. I checked the other options and the are in the XLF files just fine. :-)

Reply:
I think they are in the XLF because they are used somewhere else, but we'll see ^^

Reply:
@jolelievre I hope! If it doesn't work, VS code is messing with me. :D

![Snímek obrazovky 2024-12-04 190250](https://github.com/user-attachments/assets/686b4c86-d7ed-4864-ae2d-f4d2bfcfd11c)